hphp/tools/gdb/gdbutils.py

```diff
@@ -126,7 +126,11 @@ def string_data_val(val, keep_case=True):
     else:
         data = val['m_data']
 
-    s = data.string('utf-8', 'ignore', val['m_len'])
+    try:
+        s = data.string('utf-8', 'ignore', val['m_len'])
+    except:
+        s = "Unknown"
+
```

Comment:
Undo this change.

Reply:
@mxw - Actually, without this change the lookup fails hard and stops crawling the stack.
Example below:

#18 0xffffffffd4b0 @ 0x05c2c91c: HPHP::ObjectData::release() at /mnt/cdn/hhvm/play.083017a/hhvm/hphp/runtime/base/object-data.cpp:203
#19 0xffffacefff90 @ 0x14000228: [PHP] foo() at /mnt/cdn/hhvm/play.083017a/hhvm/hphp/test/quick/setg-dtor.php
#20 0xffffacefffc0 @ 0x0f000918: [PHP] <pseudomain>() at /mnt/cdn/hhvm/play.083017a/hhvm/hphp/test/quick/setg-dtor.php

Traceback (most recent call last):
  File "/mnt/cdn/hhvm/play.083017a/hhvm/hphp/tools/gdb/gdbutils.py", line 55, in wrapped
    return func(*args, **kwds)
  File "/mnt/cdn/hhvm/play.083017a/hhvm/hphp/tools/gdb/stack.py", line 115, in invoke
    idx=i, ar=fp.cast(ar_type), rip=rip)))
  File "/mnt/cdn/hhvm/play.083017a/hhvm/hphp/tools/gdb/frame.py", line 163, in create_php
    func_name = nameof(func)
  File "/mnt/cdn/hhvm/play.083017a/hhvm/hphp/tools/gdb/nameof.py", line 34, in nameof
    return string_data_val(deref(sd))
  File "/mnt/cdn/hhvm/play.083017a/hhvm/hphp/tools/gdb/gdbutils.py", line 130, in string_data_val
    s = data.string('utf-8', 'ignore', val['m_len'])
OverflowError: signed integer is greater than maximum

Python Exception <class 'OverflowError'> signed integer is greater than maximum: 
Error occurred in Python command: signed integer is greater than maximum